server/src/user.ts

```diff
@@ -76,8 +92,14 @@ export async function upsertUser(user: Express.User) {
  */
 export async function getUser(id: string) {
   const user = await getDb().oneOrNone<DbUser>(
-    `SELECT * FROM "user" WHERE id = $1`,
-    [id],
+    `SELECT
```

Comment:
If it's not possible to know the encryption key in the migrations, we'd need to handle any unexpected errors here in these two select queries, and maybe return some empty data. The user names and emails would then be properly filled once they log in.